lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -115,6 +119,8 @@ sub register {
   if ($config->{pipes}) {
     $self->_pipes($config->{pipes});
     $app->helper($helper => sub { @_ == 1 ? $self : $self->_render_tags(@_) });
+    $self->process(ref eq 'ARRAY' ? @$_ : $_)
```

Comment:
The reason that one place of AssetPack code init more more understandable for me.